### DISCUSSION TITLE
[firebase_messaging] getAPNSToken returns null when relaunch app

### DISCUSSION THREAD

**Discussion by Ellynnn** [firebase/flutterfire#11851] [Q&A]
I build my app in release mode for IOS and on first launch during `getAPNSToken()` it will return me a value. I killed the app and relaunch it, the `getAPNSToken()` returns null and `getToken()` will hang. Can I know if this is an expected behaviour?

However, in debug mode, `getAPNSToken()` and `getToken()` always returns a value

**Comment by alvindrakes** [comment]
Hey @Ellynnn did you managed to find the solution or cause to this problem?
